<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="DetectTab">
     <attribute name="title">
      <string>Detect Mode</string>
     </attribute>
     <widget class="QPushButton" name="CapturePicture">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Capture Picture</string>
      </property>
     </widget>
     <widget class="QLabel" name="CapResultLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>240</width>
        <height>180</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="LoadCapPic">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>270</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Original</string>
       </attribute>
       <widget class="QLabel" name="FilterLabel2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="FilterLabel4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="FilterLabel1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="FilterLabel3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Warping</string>
       </attribute>
       <widget class="QLabel" name="WarpFilterLabel1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpFilterLabel2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpFilterLabel3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpFilterLabel4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>270</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="saveResultButtom">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>220</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>SaveResult</string>
      </property>
     </widget>
     <widget class="QSlider" name="CapResultSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>200</y>
        <width>241</width>
        <height>19</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveCap">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>270</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CalibrationTab">
     <attribute name="title">
      <string>Calibration Mode</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>391</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="CalLabel2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="CalLabel4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="CalLabel3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="CalLabel1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="LoadCalButtom">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>260</y>
          <width>160</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Calibration Pic From File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="CaptureCalButtom">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>160</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Capture the Calibration Pictures</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SaveCalButtom">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>260</y>
          <width>71</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="WarpCalLabel1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpCalLabel2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>0</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpCalLabel3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="WarpCalLabel4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>160</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="CalResultLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>320</width>
          <height>240</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QListWidget" name="ProgressList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentRow">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Capture Calibration Picture</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>See The Warping Result</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>The Stitching Result of the Calibration Photo</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Black Calibration</string>
     </attribute>
     <widget class="QLabel" name="BlackReflabel1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="BlackReflabel2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="BlackReflabel3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="BlackReflabel4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>160</width>
        <height>120</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="LBlackRefButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Black Ref</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CaptureBLRef">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Capture Black Ref</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveBlackRef">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Black Ref</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="AutoDetectButtom">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Detect Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetUpButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SetUp</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
